Updates/4617_quest_7631-dreadsteed-of-xoroth-rework.sql

```diff
@@ -0,0 +1,157 @@
+-- missing emote dialog
+INSERT INTO `script_texts` (`entry`, `content_default`, `sound`, `type`, `language`, `emote`, `broadcast_text_id`, `comment`) 
+VALUES ('-1429005', '%s is pulled back to Xoroth!', '0', '2', '0', '0', '10063', 'Xoroth demon unsummoned'); 
+INSERT INTO `broadcast_text` (`Id`, `Text`, `Text1`, `ChatTypeID`, `LanguageID`, `ConditionID`, `EmotesID`, `Flags`, `SoundEntriesID1`, `SoundEntriesID2`, `EmoteID1`, `EmoteID2`, `EmoteID3`, `EmoteDelay1`, `EmoteDelay2`, `EmoteDelay3`, `VerifiedBuild`) 
+VALUES ('10063', '%s is pulled back to Xoroth!', '', '2', '0', '0', '0', '1', '0', '0', '0', '0', '0', '0', '0', '0', '31882');
+
+-- make DND creature invisible
+UPDATE `creature_template` SET `ExtraFlags` = 128 WHERE `Entry` = 14501;
```

Comment:
Faction seems to be 90, isnt there a better solution then adding 128 as extraflag?